src/styles/blocks/stars--4.css

```diff
@@ -0,0 +1,44 @@
+.stars--4 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--4 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
+}
+
+.stars--4 .stars__star:nth-child(2) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
+
+.stars--4 .stars__star:nth-child(3) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
+
+.stars--4 .stars__star:nth-child(4) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
+
+.stars--4 .stars__star:nth-child(5) {
+  display: block;
+  background-image: url(../../images/star.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
```

Comment:
There is a lot of repeated code for `.stars__star` in each child selector. Consider using a single class for the common properties to follow the DRY (Don't Repeat Yourself) principle. For example, you could have a `.stars__star` class that sets the `display`, `background-image`, `width`, `height`, and `margin-left` properties, and then only modify the `background-image` property for the fifth child if it needs to be different.